### ISSUE TITLE
Issue in data order

### ISSUE THREAD

**Issue by alehosaini** [yajra/laravel-datatables#1957]
Hi yajra,
Recently when i upgraded to newest version of Laravel 5.7
i faced an issue with data,
My eloquent query in the backend select some fields
lets say i select "id,name,contat,company"
and the data like this

id | name | contact | company
----------------------------------
1| test23 | john doe | microsoft

when i execute it in the first time in frontend side
the data comes like these
id | name | contact | company
----------------------------------
1| john doe |  | microsoft

although the select query in eloquent is correct
but when i sort the "name" field ASC the data comes correcly
id | name | contact | company
----------------------------------
1| test23 | john doe | microsoft

so in short when i sort some field in the dataTable the data comes correcly from backend
otherwise the data comes wrong.

Please help.

- Operating System XAMPP Localhost Windows
- PHP Version 7.3
- Laravel Version 5.7
- Laravel-Datatables Version 8.3
